src/pybind/mgr/dashboard_v2/controllers/rbd_mirroring.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+import re
+
+from functools import partial
+
+import rbd
+
+from dashboard_v2.controllers.dashboard import Dashboard
+from ..tools import ApiController, RESTController, ViewCache
+from .. import logger
+
+
+@ApiController('rbdmirror')
```

Comment:
add `@AuthRequired()`

Reply:
Oh, yes, of course. I left it out for testing purposes and forgot it. Thank you.